README.md

```diff
@@ -123,20 +136,22 @@ iis_virtual_directory { 'vdir':
 
 * `iis_version` - The version of the installed IIS. Empty string if not installed.
 
-### Types/Providers
+### Types and Providers
 
 * [iis_application](#iis_application)
 * [iis_application_pool](#iis_application_pool)
 * [iis_feature](#iis_feature)
 * [iis_site](#iis_site)
+* [iis_virtual_directory](#iis_virtual_directory)
+
 
 ### iis_application
 
 Allows creation of a new IIS application.
 
 #### `ensure`
 
-Must be either 'present' or 'absent'. Present will ensure the application pool is created.
+Must be either `present` or `absent`. Present will ensure the application is created.
```

Comment:
present and absent are correct in single quotes, as they are strings rather than values like `true` or `false`.